### ISSUE TITLE
NullPointerException in RconActivity

### ISSUE THREAD

**Issue by daparker** [daparker/checkvalve#7]
A crash was reported in the RCON Activity.  The stack trace is:

```
java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String java.lang.Object.toString()' on a null object reference
    at com.dparker.apps.checkvalve.RconActivity$3.handleMessage(RconActivity.java:151)
    at android.os.Handler.dispatchMessage(Handler.java:102)
    at android.os.Looper.loop(Looper.java:135)
    at android.app.ActivityThread.main(ActivityThread.java:5307)
    at java.lang.reflect.Method.invoke(Native Method)
    at java.lang.reflect.Method.invoke(Method.java:372)
    at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:904)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:699)
    at de.robv.android.xposed.XposedBridge.main(XposedBridge.java:115)
```

The offending code is:

`Log.d(TAG, "Message object string = " + msg.obj.toString());`

Clearly, a null value check for `msg.obj` is required.

**Comment by daparker** [comment]
This occurs when then engine query returns a value of `-1`.  The message object will always be null in this case, so the log lines which reference `msg.obj` should simply be removed.

**Comment by daparker** [comment]
Turns out this was already fixed in the 2.0-development branch (commit [1c4c63d](https://github.com/daparker/checkvalve/commit/1c4c63d4d57cdacb9f5a91e3cb25de3ce152c0ff#diff-b1796e79301975dcdb872a85044631d7)).  The fix will be pushed to the master branch in the next release.

**Comment by daparker** [comment]
Fixed in version 2.0.5 (commit [34af762](https://github.com/daparker/checkvalve/commit/34af762336dfbcff2ba9fb219062227ffbd6298c))
